docs/index.rst

```diff
@@ -40,6 +40,7 @@ and the vineyard store that offers efficient in-memory data transfers.
    python_tutorials
    java_tutorials
    cpp_tutorials
+   gremlin_tutorials
```

Comment:
Actually, I don't think this file is like a tutorial:
A tutorial is more like a step-by-step guide and wrapped up as a story/process.

For example, if there will be a `gremlin_tutorial` here, 
I would expect it covers these sections:
- a brief introduction to Gremlin;
- Launch the GAIA;
- loading a graph data (maybe a drawing visualizing the graph)
- supported gremlin-client, how to connect the server;
- connected, several gremlin queries as examples, full steps please refer to THIS file

conclusively, it maybe more like 
[The First Five Minutes](https://tinkerpop.apache.org/docs/current/tutorials/getting-started/) or [The Next Fifteen Minutes](https://tinkerpop.apache.org/docs/current/tutorials/getting-started/) in the official doc. 


This file is more like an indexing page listing all available/unavailable steps.
Thus, I suggest rename this file as "Supported Gremlin Steps" under ADVANCED TOPICS
and below 
- dive into engines (which will expand to each engine, as below)
- analytical engine... (To appear) 
- GAIA: interactive engine (To appear) 
- Supported Gremlin Steps

How do you think?